meta-mender-core/recipes-mender/mender-artifact/mender-artifact_3.1.1.bb

```diff
@@ -0,0 +1,25 @@
+require mender-artifact.inc
+
+################################################################################
+#-------------------------------------------------------------------------------
+# THINGS TO CONSIDER FOR EACH RELEASE:
+# - SRC_URI (particularly "branch")
+# - SRCREV
+# - DEFAULT_PREFERENCE
+#-------------------------------------------------------------------------------
+
+SRC_URI = "git://github.com/mendersoftware/mender-artifact.git;protocol=https;branch=3.1.x"
+
+# Tag: 3.1.1
+SRCREV = "1bdd3da8203ec274ad4ff9d0bc6de7907372b074"
+
+# Enable this in Betas, not in finals.
+# Downprioritize this recipe in version selections.
+#DEFAULT_PREFERENCE = "-1"
```

Comment:
Remember to uncomment all of these. Mender >= 2.0 is not enabled by default on thud because it is backwards incompatible.

Reply:
Right! Thanks

Reply:
And make sure this gets cherry-picked to older branches as well.